lib/preface_make/category.ml

```diff
@@ -1,25 +1,11 @@
 module Core (Req : Preface_specs.Category.WITH_ID_AND_COMPOSE) = Req
```

Comment:
I think that it could be good to add `Category.Over_semigroupoid`. What do you think about that?

Reply:
In case we must specify the following module in Category:
```ocaml
(** Minimal interface using [id]. *)
module type WITH_ID = sig
  type ('a, 'b) t
  (** The type held by the [Category]. *)

  val id : ('a, 'a) t
  (** The identity morphism. *)
end
```